### PR TITLE
[jwt-cpp] provide cmake config files for find package

### PR DESCRIPTION
**Pull request by prince-chrismc**
If this PR updates an existing port, please uncomment and fill out this checklist:

- [x] Changes comply with the [maintainer guide](https://github.com/microsoft/vcpkg-docs/blob/main/vcpkg/contributing/maintainer-guide.md)
- [x] SHA512s are updated for each updated download
- [x] The "supports" clause reflects platforms that may be fixed by this new version
- [x] Any fixed [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt) entries are removed from that file.
- [x] Any patches that are no longer applied are deleted from the port's directory.
- [x] The version database is fixed by rerunning `./vcpkg x-add-version --all` and committing the result.
- [x] Only one version is added to each modified port's versions file.

### CODE REVIEW

**Comment by prince-chrismc** [comment]
It's very hard to follow the longs for Azure 🙊 I think it's not finding openssl?

How do I make that feature dependencies on by default? Reading it's not obvious https://learn.microsoft.com/en-us/vcpkg/reference/vcpkg-json#dependency

**Comment by JonLiu1993** [comment]
Duplicate of PR #36916.

**Comment by prince-chrismc** [comment]
That PR doesn't fix the cmake config files not be available for find package

**Comment by JonLiu1993** [comment]
> That PR doesn't fix the cmake config files not be available for find package

Yes, but it has been merged. Could you change the title of your PR and fix this problem in this PR? thanks.
